app/models/bulkrax/importer.rb

```diff
@@ -153,11 +153,11 @@ def import_metadata_format
 
     # If the import data is zipped, unzip it to this path
     def importer_unzip_path
-      @importer_unzip_path ||= File.join(Bulkrax.import_path, "import_#{path_string}")
+      @importer_unzip_path ||= File.join(Bulkrax.base_path.call('import'), "import_#{path_string}")
```

Comment:
becomes:

```
@importer_unzip_path ||= File.join(parser.base_path('import'), "import_#{path_string}")

Reply:
ahhh. this is probably what I was missing in my first attempt. I was calling the method directly instead of from the parser. 
thanks!